### PR TITLE
fix(prepush): adjusts pre-commit to run tests in shell

### LINKED ISSUE #201: Add pre-push precommit instructions to README.md
**Acceptance Criteria:**

- [ ] Instructions for running the following line are added to [README.md](http://readme.md/) under Submitting Code > Pre-Commit Setup:
`pre-commit install --hook-type pre-push`
`pre-commit installed at .git/hooks/pre-push`

### PR DESCRIPTION
**Pull request by esparr**
### **1. Targeted Issue** ###

[Issue #167](https://trello.com/c/FEvL708r)
[Issue #169](https://trello.com/c/cA85T9PN)

### **2. Overview of Solution** ###

This PR addresses two tickets, see above. 

I added in the pre-push documentation and also am offering up a solution to demonstrating outputs without compromising the ability to easily copy code from the README.md on Github. I am open to feedback on the implementation. 

This also solves an issue I was experiencing with Github Desktop when running pre-push before pushing up to Github. The solution was to make sure the tests were running in pipenv shell. @jimbandrews assisted with this solution

**Other changes:**
Cleans up lingering linting issues

### **3. Tools Used** ###

`pipenv run`

### **4. Testing Strategy** ###

Tried to run this with Github Desktop and from shell in terminal. Both successfully ran the tests. 

### **6. Screenshots** ###

Preview mode:
Screen Shot 2023-11-02 at 3 04 22 PM

Markdown:
Screen Shot 2023-11-02 at 3 05 00 PM

### **7. Code Reviewers** ###

@jimbandrews @GitLukeW 

---

Before you go...

- [x]  I have linked the relevant issue
- [x]  I have provided an overview of my solution
- [x]  I have shared the tools I've used
- [x]  I have outlined my testing strategy
- [x]  I have considered future implications of my changes
- [x]  I have included relevant screenshots
- [x]  I have kept my team in the loop

### CODE REVIEW

File: Pipfile.lock
Comment (esparr) [inline Pipfile.lock:1]: @GitLukeW During the process of debugging, I uninstalled and reinstalled celery. When I did this, I also ran pipenv install and it looks like it updated boto3 as well. Do you think this is going to cause an issue? If so I can work on reverting it.

**Review by jimbandrews — APPROVED** [review]
Tested out locally, and it works. Good catch!

**Review by GitLukeW — CHANGES_REQUESTED** [review]
Looks like this has merge conflicts from the previous PR.

**Review by GitLukeW — APPROVED** [review]
Thanks!

**Pull request merged**